doc/md/cloud/getting-started.md

```diff
@@ -0,0 +1,220 @@
+to your GitHub repository, you will need to setup the Atlas GitHub action on your repository by following these steps:
+
+:::note
+If you already have the Atlas GitHub action set up, you may skip step 4. In step 5, only add
+`ariga-token: ${{ secrets.ARIGA_TOKEN }}` to your yaml file.
+:::
+
+1. From the Settings page, generate an access token under 'Tokens'.
+2. On your GitHub repo, under the 'Settings' section, click on 'Secrets' > 'Actions' to create a new repository secret.
+3. Name your secret (for example, ARIGA_TOKEN) and paste the generated token from step 2.
+4. Install the Atlas GitHub Action by adding a file named `.github/workflows/atlas-ci.yaml` to your repo.
+5. Based on the type of database you are using, copy the following code into the workflow definition file. Set up
+the `ariga-token` input parameter to the secret name you chose in the previous step, and ensure your
+mainline branch and migration directory path are configured correctly:
+
+<Tabs
+defaultValue="mysql"
+values={[
+{label: 'MySQL', value: 'mysql'},
+{label: 'PostgreSQL', value: 'postgres'},
+{label: 'MariaDB', value: 'maria'},
+]}>
+<TabItem value="mysql">
+
+```yaml {7,11,36,39}
+name: Atlas CI
+on:
+  # Run whenever code is changed in the master branch,
+  # change this to your root branch.
+  push:
+    branches:
+      - master
+  # Run on PRs where something changed under the `path/to/migration/dir/` directory.
+  pull_request:
+    paths:
+      - 'path/to/migration/dir/*'
+jobs:
+  lint:
+    services:
+      # Spin up a mysql:8.0.29 container to be used as the dev-database for analysis.
+      mysql:
+        image: mysql:8.0.29
+        env:
+          MYSQL_ROOT_PASSWORD: pass
+          MYSQL_DATABASE: test
+        ports:
+          - "3307:3306"
+        options: >-
+          --health-cmd "mysqladmin ping -ppass"
+          --health-interval 10s
+          --health-start-period 10s
+          --health-timeout 5s
+          --health-retries 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.1
+        with:
+          fetch-depth: 0 # Mandatory unless "latest" is set below.
+      - uses: ariga/atlas-action@latest
+        with:
+          dir: path/to/migrations
+          dir-format: atlas # Or: golang-migrate, goose, dbmate
+          dev-url: mysql://root:pass@localhost:3307/test
+          ariga-token: ${{ secrets.ARIGA_TOKEN }}
+```
+
+</TabItem>
+<TabItem value="postgres">
+
+```yaml {7,11,35,38}
+name: Atlas CI
+on:
+  # Run whenever code is changed in the master branch,
+  # change this to your root branch.
+  push:
+    branches:
+      - master
+  # Run on PRs where something changed under the `path/to/migration/dir/` directory.
+  pull_request:
+    paths:
+      - 'path/to/migration/dir/*'
+jobs:
+  lint:
+    services:
+      # Spin up a postgres:10 container to be used as the dev-database for analysis.
+      postgres10:
```

Comment:
PostgreSQL 10 is EOL in November (2 Month from now): Our example should be "fresher", I think :-)